src/app/shared/components/NameForm/index.tsx

```diff
@@ -55,11 +63,22 @@ const NameForm = (props: NameFormProps) => {
           <div className="text-xl"> 
             {title}
           </div>
-          
-          <FontAwesomeIcon
-            icon={faEdit}
-            onClick={() => setIsEditing(true)}
-          />
+
+          <div className="flex items-center">
+            <FontAwesomeIcon
+              className="cursor-pointer"
+              icon={faEdit}
+              onClick={() => setIsEditing(true)}
+            />
+
+            <FontAwesomeIcon
```

Comment:
this needs a little work since its unclear what is being copied and there is no feedback
should we be providing a link here or what?

Reply:
Yep let me handle that in my pages ticket